src/Elm/Parser/Expression.elm

```diff
@@ -753,11 +753,11 @@ negationOperation =
                         (Node { start, end } _) =
                             subExpressionResult.syntax
                     in
-                    { comments = subExpressionResult.comments
-                    , syntax =
-                        Node
-                            { start = { row = start.row, column = start.column - 1 }, end = end }
-                            (Negation subExpressionResult.syntax)
+                    { subExpressionResult
```

Comment:
A record update expression is slower than creating a new one. A performance tip is usually to try to have the former code rather than the record update.

Reply:
Thought there was an optimization for that in eol2. Fixed.

Reply:
I think you're thinking of https://blogg.bekk.no/improving-the-performance-of-record-updates-cb34cb6d4451

I think it's better in eol2 compared to without, but I would imagine it's still slower than a plain record definition (if the Elm compiler used opportunistic mutation, maybe then...)